pyproject.toml

```diff
@@ -1,14 +1,16 @@
 [build-system]
-requires = ["setuptools>=64"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
 [project]
 name = "doge"
 version = "3.8.0"
 description = "wow very terminal doge"
 authors = [{ name = "Olivia Thiderman", email = "olivia@thiderman.org" }]
+urls = { Homepage = "https://github.com/thiderman/doge" }
```

Comment:
This is my old username.  Maybe switch to the current one? :slightly_smiling_face:

Reply:
Of course. Fixed!